### ISSUE TITLE
Incompatible with einops

### ISSUE THREAD

**Issue by CreamNuts** [rentruewang/aioway#2]
Thanks for your nice project. But it is not compatible with [einops](https://github.com/arogozhnikov/einops). 
Einops is convenient for pytorch user, and I think many people use it. 
I hope it works on einops, too. Thank you.

**Comment by rentruewang** [comment]
Good point. I'll be working on it. Feel free to add more feature requests!

**Comment by rentruewang** [comment]
Ok. After quickly skimming over `einops`'s source code, I noticed that they call `torch.*` functions behind the scenes. That means their functions will directly work with Koila, because Koila uses PyTorch's extension API.

With that being said, not all `torch.*` functions are implemented yet, and it probably requires some testing. So I'll leave it open for now, and see what I can do when I have some time.

**Comment by CreamNuts** [comment]
I'm looking forward to implementing it as soon as possible.

**Comment by rentruewang** [comment]
Closing in favor of #39
